less/dropdowns.less

```diff
@@ -212,6 +231,20 @@
   right: auto;
 }
 
+@media (max-width: @screen-sm-min) {
+  .dropdown-menu-native {
+    position: fixed;
+    bottom: 0;
+    top: auto;
+    width: 100%;
+    max-width: 100% !important;
+    max-height: 50% !important;
```

Comment:
Not sure this is working:
![image](https://user-images.githubusercontent.com/1318111/38895915-3f2c2550-4289-11e8-8b00-ece28baa716c.png)